README.md

```diff
@@ -45,3 +45,12 @@ Then run the api:
 
 #### How to run the api with the DB
 `make start-api`
+
+#### Frontend
+
+To run the frontend you should jump to the ./frontend folder using the command cd ./frontend and run the following command:
+
+`npm run dev` or
+`yarn dev`
```

Comment:
we could have kept the `npm install` step as well. 
Also it is recommended to use npm as we don't have a `yarn.lock` file in the folder